### PR TITLE
Update jackson-databind to 2.8.11.6

### PR DESCRIPTION
**Pull request by jasontedor**
This commit upgrades the jackson-databind depdendency to 2.8.11.6. Additionally, we revert a previous change that put ingest-geoip on the version of jackson-databind from the version properties file. This is because upgrading ingest-geoip to a later version of jackson-databind also requires an upgrade to the geoip2 dependency which is currently blocked. Therefore, if we can get to a point where we otherwise upgrade our Jackson dependencies, we do not want ingest-geoip to automatically come along with it.

Relates #45225

### CODE REVIEW

File: modules/ingest-geoip/build.gradle [hunk 1]
```diff
@@ -28,8 +28,8 @@ dependencies {
   // Upgrade to 2.10.0 or higher when jackson-core gets upgraded to 2.9.x. Blocked by #27032
   compile('com.maxmind.geoip2:geoip2:2.9.0')
   // geoip2 dependencies:
-  compile("com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}")
-  compile("com.fasterxml.jackson.core:jackson-databind:${versions.jacksondatabind}")
+  compile("com.fasterxml.jackson.core:jackson-annotations:2.8.11")
+  compile("com.fasterxml.jackson.core:jackson-databind:2.8.11.6")
```
Comment (dakrone) [inline modules/ingest-geoip/build.gradle]: Should these still use the `${versions.jackson}` (etc) substitution instead of being hardcoded?

Comment (jasontedor) [reply modules/ingest-geoip/build.gradle]: I explained in the commit message why I made this change. See also #53523.

Comment (dakrone) [reply modules/ingest-geoip/build.gradle]: Doh, sorry, yes I totally forgot to read that.

**Review by dakrone — APPROVED** [review]
LGTM, do you think we should add a comment where you hardcoded the version in the ingest-geoip build just so no one reverts the change?

**Comment by jasontedor** [comment]
> LGTM, do you think we should add a comment where you hardcoded the version in the ingest-geoip build just so no one reverts the change?

Yes, that sounds like a good idea. I thought the comment on version.properties would be enough.

**Pull request merged**